.gitignore

```
@@ -2,6 +2,12 @@
 .gradle
 build
 out
+application-prod.yml
+datasource-prod.yml
+application-local.yml
+datasource-local.yml
+naver-cloud-platform-auth.yml
```

```suggestion
application-prod.yml
datasource-prod.yml
application-local.yml
datasource-local.yml
naver-cloud-platform-auth.yml
```

몇 개는 들어가면 안될 것 같긴하지만, 결국은 다 들어가야하지 않을까 싶고, 시크릿들을 어떻게 처리해야할지는 따로 고민해야할 것 같습니다. :-)